### PR TITLE
Refactor out javascript from php views

### PR DESCRIPTION
**Pull request by godDLL**
As previously discussed in #11

### CODE REVIEW

**Comment by godDLL** [comment]
In the process we lost support for your Shortcut plugin.

File: views/buttons.php [hunk 1]
```diff
@@ -14,33 +14,7 @@
 </div>
 </div>
 <script type="text/javascript" charset="utf-8">
-$().ready(function() {
-    var target = this.hash;
```
Comment (godDLL) [inline views/buttons.php:18]: anyone remember what this was for?  thought so

**Comment by godDLL** [comment]
I think I'll do screenshots, some README additions, a version bump to 0.2.1 maybe?
A little furnishing and the plugin becomes _more welcoming_, which means more users.

**Comment by godDLL** [comment]
Currently the Website link in the Administration panel is not usable. That's a bug mate.
No website, scant README, that's no way to go.

**edit**: not that my own website linked in profile is in any kind of shape yet, but you know, point stands

File: views/buttons.php (hunk 1 above)
Comment (svanlaere) [inline views/buttons.php:18]: I have no idea, I borrow/steal other peoples javascript to make things work ;)

**Comment by godDLL** [comment]
The only way for people to reliably find out about your plugin is
https://www.wolfcms.org/repository/list/ and it's not on there.

**Comment by svanlaere** [comment]
This plugin is to be found at https://www.wolfcms.org/repository/202

**Comment by svanlaere** [comment]
Screenshots and version bump is fine with me, makes sence too.

**Comment by godDLL** [comment]
Also a CONTRIBUTING file, with "If you encounter a bug, or have an idea for improvement - please make a new issue; if your bug or idea comes with working code, please make a pull request onto :master".

For many people the project's licensing stance will be important. Should go in the README.

**Comment by godDLL** [comment]
> This plugin is to be found at https://www.wolfcms.org/repository/202

That's funny, because if I search https://www.wolfcms.org/repository/list/ for "snippet":
_1 result, "Shortcut", which is your other plugin_

If I then click on the 'snippet' tag (which makes a different kind of search), I get:
_1 result, "Shortcut"_

If I instead search for "easysnippet" or "Easysnippet" or even "easy snippet" I get _nada_.

Going over the https://www.wolfcms.org/repository/list/ page by page, I _can't find it_.
Only _occasionally_ it crops up in the "Latest Plugins" section in the footer, for whatever reason.

Really, the only way to get to that page, since it's _not linked_ in the README and the plugin doesn't have a website, is with a Google search like `site:www.wolfcms.org Easysnippet`

**Comment by svanlaere** [comment]
Well I guess the README needs a link, but indeed your correct its not listed.
I'll have to ask Martijn about this.

**Comment by svanlaere** [comment]
Found out what is wrong, I have to publish the plugin.
Now it gets listed.

**Comment by godDLL** [comment]
OK, then I don't so much care about the bug where if you list all my plugins on the Wolf repository (none), you get ALL of them.

**Comment by svanlaere** [comment]
I didn't even realise lol
Its a old plugin and probally removed it at one point to later add it again.
As for tags, I can't add those that is up to Martijn I think.

**Comment by godDLL** [comment]
Let's ping him about that. Can you add screenshots and links and a better description to the plugin repo page though? I'll make screenshots, and take care of the README and all that.

**Comment by svanlaere** [comment]
Yeah, well the plugin can at least now be found  :+1:

**Pull request merged**